README.md

```diff
@@ -74,12 +86,23 @@ In the code snippet below we want text entered into the input field to be displa
 ```js
 // assume all necessary components are imported above
 class App extends Component {
+  constructor() {
+    super()
+    this.state {
+      value: ''
+    }  
+    this.handleChange = this.handleChange.bind(this);      
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
   render() {
     return (
       <div>
         <form>
           <label>Input: </label>
-          <input type="text" />
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
         </form>
         <p>Message: { this.state.message }</p>
```

Comment:
Since you're using the state variable `value` in your code additions, `message` should have been changed to `value` here to match.